src/main/java/org/opentestsystem/ap/ims/service/impl/VersionedItemServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package org.opentestsystem.ap.ims.service.impl;
+
+import org.opentestsystem.ap.common.model.Item;
+import org.opentestsystem.ap.ims.model.ItemResource;
+import org.opentestsystem.ap.ims.service.VersionedItemService;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class VersionedItemServiceImpl implements VersionedItemService {
+    @Override
+    public Optional<Item> getItem(final String itemId, final String version) {
+        return Optional.empty();
+    }
+
+    @Override
+    public Optional<ItemResource> getItemResource(final String itemId, final String version, final String fileName) {
+        return Optional.empty();
+    }
+}
```

Comment:
And this is the service we need your help with. Thank you in advance!

Reply:
This makes sense given the conversation yesterday.  It is more clear what I'll need to do.